src/components/new-conversation-dialog/create-room/theme.js

```diff
@@ -2,8 +2,12 @@ export default {
   wrapper: {
     flex: 1,
   },
-  link: {
+  backWrapper: {
     flex: '0 0 auto',
+    display: 'flex',
+  },
+  back: {
```

Comment:
backButton would be better I guess

Reply:
but the class is on a div (containing the button and the text), still call it backButton? but we don't need to discuss it if we change it to an anchor anyway 👍